### PR TITLE
Added fallthrough_on_empty field for match object

### PR DESCRIPTION
**Pull request by Mixaster995**
Signed-off-by: Mikhail Avramenko <avramenkomihail15@gmail.com>

## Issue
part of https://github.com/networkservicemesh/deployments-k8s/issues/2007

### CODE REVIEW

**Review by denis-tingaikin — CHANGES_REQUESTED** [review]
Please re-generate proto models.

File: pkg/api/registry/registry.proto [hunk 1]
```diff
@@ -15,6 +15,7 @@ message NetworkService {
 message Match {
     map<string, string> source_selector = 1;
     repeated Destination routes = 2;
+    bool fallthrough_on_empty = 3;
```
Comment (denis-tingaikin) [inline pkg/api/registry/registry.proto:18]:
```suggestion
    bool  fallthrough = 3;
```

Comment (Mixaster995) [reply pkg/api/registry/registry.proto]: ok, done

File: pkg/api/registry/registry.pb.go [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.14.0
```
Comment (denis-tingaikin) [inline pkg/api/registry/registry.pb.go:4]: Please use these versions of generating tools to be consistent with other models.
Updating for new versions should be done in separate PR.

Comment (Mixaster995) [reply pkg/api/registry/registry.pb.go]: done

**Review by denis-tingaikin — APPROVED** [review]
LGTM

**Comment by denis-tingaikin** [comment]
@edwarnicke Can we merge this PR?
